examples/models/neurons/bipolar_cell.py

```diff
@@ -20,33 +20,34 @@
 # along with DeNSE. If not, see <http://www.gnu.org/licenses/>.
 
 
-""" Generate the morphology of a bipolar cell (spindle neuron) """
+""" Generate the morphology of a bipolar cell (spindle neuron) 
+    Also illustrates generation of dendogram
+"""
 
 import numpy as np
 import os
 
 import matplotlib.pyplot as plt
 
+import neurom as nm
```

Comment:
did we not say that we would encapsulate all parts requiring ``neurom`` into

```python
try:
    import neurom
    ...
except ImportError:
    pass
```

therefore doing the neurom import at the end of the file?